Objects/genobject.c

```diff
@@ -1031,7 +1011,8 @@ static PyMethodDef coro_methods[] = {
 static PyAsyncMethods coro_as_async = {
```

Comment:
Wait, with this PR regular generators no longer benefit from the faster `PyIter_Send`. I think we should implement `am_send` for regular generators too.